projects/iio_demo/src/hal/main.h

```diff
@@ -0,0 +1,83 @@
+/* USER CODE BEGIN Header */
```

Comment:
nitpick: should main.h be in the "hardware abstraction layer" folder? I'm thinking it should go into the app folder with it's .c counterpart.

Reply:
Yes, it looks weird, but unfortunately that's what STM32CubeIDE creates, and I think it's easier for the user to simply move the files to the hal/ directory without renaming.